celestia.go

```diff
@@ -110,14 +136,29 @@ func (c *CelestiaDA) Validate(ids []da.ID, daProofs []da.Proof) ([]bool, error)
 		proofs = append(proofs, proof)
 	}
 	for i, id := range ids {
-		// TODO: extract commitment from ID
-		isIncluded, err := c.client.Blob.Included(c.ctx, c.height, share.Namespace(c.namespace), proofs[i], blob.Commitment(id))
-		if err != nil {
-			return nil, err
-		}
+		height, commitment := splitID(id)
+		// TODO(tzdybal): for some reason, if proof doesn't match commitment, API returns (false, "blob: invalid proof")
+		//    but analysis of the code in celestia-node implies this should never happen - maybe it's caused by openrpc?
+		//    there is no way of gently handling errors here, but returned value is fine for us
```

Comment:
> maybe it's caused by openrpc

Don't think we're using openrpc.

Reply:
celestia-node `api/rpc` is using openrpc.